tools/memory_estimator/memory_estimator.py

```diff
@@ -20,36 +20,40 @@
 
 __FREERTOS_SRC_DIR__ = os.path.join('FreeRTOS', 'Source')
 __FREERTOS_PLUS_SRC_DIR__ = os.path.join('FreeRTOS-Plus', 'Source')
-__IOT_LIBS_DIR__ = os.path.join(__FREERTOS_PLUS_SRC_DIR__, 'FreeRTOS-IoT-Libraries-LTS-Beta1')
+__IOT_LIBS_BETA1_DIR__ = os.path.join(__FREERTOS_PLUS_SRC_DIR__, 'FreeRTOS-IoT-Libraries-LTS-Beta1')
+__IOT_LIBS_BETA2_DIR__ = os.path.join(__FREERTOS_PLUS_SRC_DIR__, 'FreeRTOS-IoT-Libraries-LTS-Beta2')
 
 __LIB_NAME_TO_SRC_DIRS_MAPPING__ = {
     'light-mqtt' : [
```

Comment:
Do we want `light-mqtt-beta1` and `light-mqtt-beta2`?

Reply:
As we do not plan to have an equivalent FreeRTOS.org page for the Beta2 version of Lightweight MQTT under FreeRTOS Labs as the Beta1 version (existing page: https://freertos.org/mqtt/preconfiguredexamplesLW.html), we wouldn't need the `light-mqtt-beta2` entry..